### PR TITLE
Add Tb3py

### PR DESCRIPTION
**Pull request by knc6**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by knc6** [comment]
@conda-forge/staged-recipes This is ready for review

**Comment by knc6** [comment]
@conda-forge/staged-recipes This is ready for review

File: recipes/tb3py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tb3py-{{ version }}.tar.gz
+  sha256: 6105173883cd6cbf8544413aedf5e5aa4f47bcaa62be9a8ec8c94a951345f9cd
+
+build:
+  skip: true  # [win]
+  number: 0
+#   noarch: python
```
Comment (dopplershift) [inline recipes/tb3py/meta.yaml:16]: If the only reason this is not noarch and is skipping windows is due to the missing dependency, we can merge this as noarch and ignore the CI failure. The noarch packages are only currently built on linux, and the missing dependency will just result in the package not being installable on Windows.

Comment (knc6) [reply recipes/tb3py/meta.yaml]: Hi @dopplershift 

Thanks for the review and comments.

I don't necessarily want to use `noarch: python`, the usual one is fine for me. As all the tests are already passing right now, could you please review to merge? In future if it causes any issue, I'll modify the tb3py-feedstock.

Comment (dopplershift) [reply recipes/tb3py/meta.yaml]: Well noarch Python is our "usual" for pure-Python packages. So can you explain more the reasoning behind your hesitance making this noarch?

Comment (knc6) [reply recipes/tb3py/meta.yaml]: @dopplershift  Ok, using noarch-python now, please take a look.

File: recipes/tb3py/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tb3py-{{ version }}.tar.gz
+  sha256: 6105173883cd6cbf8544413aedf5e5aa4f47bcaa62be9a8ec8c94a951345f9cd
+
+build:
+  skip: true
```
Comment (dopplershift) [inline recipes/tb3py/meta.yaml:14]: This is telling conda-build to skip every build:
```suggestion
```

File: recipes/tb3py/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - pip
+    - python >=3.6
+    - requests
+  run:
+    - jarvis-tools
+    - julia
+    - numpy
+    - pytest
+    - python >=3.6
+    - requests
```
Comment (dopplershift) [inline recipes/tb3py/meta.yaml:29]: Any reason not to match the version numbers in the [upstream repo](https://github.com/usnistgov/tb3py/blob/9b0f230146b919c27a9247fdf530ba6a3a4d0e25/setup.py#L54-L58)?

**Review by dopplershift — COMMENTED** [review]
We can ignore the Windows build failure since it was due to a missing dependency (julia) and this is noarch.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**